### ISSUE TITLE
Refactor achievements to arise from Membership

### ISSUE THREAD

**Issue by philbarker** [pesc-org/json-ld-transcript#57]
As mentioned in #52, in CEDS Ontology the Membership class serves many of the roles of the properties from the Achievement super class.

So looking for some way to say 

```mermaid
graph LR
P(Person) -- hasMembership --> M(Membership) -- hasPostSecondaryProgram --> PSP(PostSecondaryProgram)
M -- hasPostsecondaryStudentAcademicAward --> PSAA(PostsecondaryStudentAcademicAward)
```

Trouble is that in CEDS I can find hasPostSecondary**Student**Program not hasPostSecondaryProgram

**Comment by philbarker** [comment]
From Nathan Clinton (@aemnathanclinton):
> For the situation you describe below, I think we do have in the next version of the Ontology what you're looking for, but through a slightly different route.  It's more like this: 


```mermaid
graph LR
P(Person) -- hasMembership --> M(Membership) -- hasOrganization --> Pr(Program)
M -- hasPostsecondaryStudentAcademicAward --> PSAA(PostsecondaryStudentAcademicAward)
```

>  We are proposing that :Program become a subclass of Organization in PR [46](https://github.com/CEDStandards/CEDS-Ontology/pull/46).

**Comment by philbarker** [comment]
We decided on the call this week that factoring in the CEDS membership might change the structure of the model so much that is would be a de facto change in the standard, something we are trying not to do. 
We agreed to use as much of CEDS as we could without introducing Memberships in order to link the "achievements" to the courses/programs/tests etc. from which they arise.

Closing this issue as it deals exclusively with refactoring to use ceds:Membership
